config/schedule.rb

```diff
@@ -0,0 +1,11 @@
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+every 2.hours do
+  # command "/usr/bin/some_great_command"
+  # runner "MyModel.some_method"
+  # rake "some:great:rake:task"
+end
```

Comment:
This is why people get addicted to ruby and, specifically, to Ruby on Rails.  I don't know if you guys have had to deal with setting up chron jobs before, but this is an INSANELY nice dsl for writing chron jobs.  Like I said in the PR description, there will be more added to this after we touch base on the weather api methods that are currently in another branch.